source/frontend/src/features/leases/add/LeaseDetailSubForm.tsx

```diff
@@ -207,6 +207,7 @@ export const LeaseDetailSubForm: React.FunctionComponent<ILeaseDetailsSubFormPro
           formikProps={formikProps}
           field="startDate"
           required={statusTypeCode === ApiGen_CodeTypes_LeaseStatusTypes.ACTIVE}
+          popperPlacement="top-end"
```

Comment:
Question: I think this story calls for placing the calendar popup on the top-end for all date fields. Did you check across all forms in PIMS that they don't need updates like the ones here? 
An alternative would be to make "top-end" the default value of the "popperPlacement" property so all datepickers support it by default

Reply:
@asanchezr, thanks for the heads up. I changed the code to apply the changes at the Datepicker object level instead.